src/libs/isReportMessageAttachment.js

```diff
@@ -8,5 +8,6 @@ import CONST from '../CONST';
  * @returns {Boolean}
  */
 export default function isReportMessageAttachment({text, html}) {
-    return text === CONST.ATTACHMENT_MESSAGE_TEXT && html !== CONST.ATTACHMENT_MESSAGE_TEXT;
+    const regex = new RegExp(` ${CONST.ATTACHMENT_SOURCE_ATTRIBUTE}="(.+?)"`, 'i');
```

Comment:
```suggestion
    const regex = new RegExp(` ${CONST.ATTACHMENT_SOURCE_ATTRIBUTE}="(.*)"`, 'i');
```
So this is what it's using on the backend, if we want it to be exactly the same. Open to discussion though.

Reply:
@dangrous Thanks for looking into this.
I think the difference between `(.*)` and `(.+?)` is that the former is greedy and the latter doesn't allow empty. See below regex test
<img width="400" alt="image" src="https://github.com/Expensify/App/assets/117511920/e7264cb6-457c-4c15-b0c9-061727dd715c">
<img width="400" alt="image" src="https://github.com/Expensify/App/assets/117511920/4af2d638-e4fa-42b2-972e-fb241e4149ad">

Actually we rely more on the first part of the regex ` data-expensify-source="` to check if the special property is included in the html string. We don't need capture the exact url here, so I think both `(.*)` and `(.+?)` will work but `(.+?)` looks more exact.

Reply:
@rushatgabhane Thanks for looking into this! I also agree, so let's keep same with backend.